router/doctors.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const router = express.Router();
+
+const User = require("../models/userModel");
+
+router.get("/", async (req, res) => {
+  try {
+    const doctors = await User.find({ role: "doctor" })
+  .select("firstName lastName email role  specialty")
+  .exec();
+
+    res.status(200).json(doctors);
+  } catch (error) {
+    res.status(404).json({ message: error });
```

Comment:
You can add the error message as a string inside the object, beacuse now if there is an error { message: "" } will be displayed.